openresty/Dockerfile

```diff
@@ -36,4 +36,4 @@ RUN mkdir -p /usr/local/openresty/nginx/client_body_temp \
 # upgrade apk packages
 RUN apk upgrade --no-cache
 
-USER nobody
+USER 65534
```

Comment:
what does this number correspond to? Is it a random static id we are assigning?

Reply:
This is default nobody user available in docker image

```nobody:x:65534:65534:nobody:/:/sbin/nologin``` - since our securityContext is set to runAsNonRoot: true it now needs uid not username